app/services/create_new_device_alert.rb

```diff
@@ -0,0 +1,45 @@
+class CreateNewDeviceAlert < ApplicationJob
```

Comment:
Can we prevent this code from running / querying for users until the feature flag is enabled?

Reply:
I see an example of that in job_configurations and have added that there.